### DISCUSSION TITLE
How can I use `Caps Lock` to exit Vim insert mode on Linux?

### DISCUSSION THREAD

**Discussion by TobisLee** [zed-industries/zed#17054] [Feature Requests]
Sorry for the silly question, but I've searched the issues and docs and couldn’t find the answer.
I want to use `Caps Lock` to exit insert mode in Zed on Linux, but I don't want to remap `Caps Lock` to `Escape` using `setxkbmap`. Is there a way to represent `Caps Lock` in the Zed keymap?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
